_layouts/eip.html

```diff
@@ -42,3 +42,22 @@ <h1 class="page-heading">
   {{ content }}
 
 </div>
+{% comment %}
+Article schema specification:
+https://schema.org/Article
```

Comment:
I wonder why was my comment about the schema ignored in the old PR: https://github.com/ethereum/EIPs/pull/2738#discussion_r456624968

I suggested `TechArticle` with more fields.

See `schema.org` for the distinction:
> TechArticle
> Thing > CreativeWork > Article > TechArticle
> A technical article - Example: How-to (task) topics, step-by-step, procedural troubleshooting,  specifications, etc.

vs.

> Article
> Thing > CreativeWork > Article
> An article, such as a news article or piece of investigative report. Newspapers and magazines have articles of many different types and this is intended to cover them all.
>
> See also blog post.

I do not think EIPs should be considered as blog posts or news articles.